Detectors/MUON/MCH/Raw/Encoder/Digit/DigitReader.cxx

```diff
@@ -0,0 +1,73 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "DigitReader.h"
+#include <TFile.h>
+#include <TTree.h>
+#include <TBranch.h>
+#include <limits>
+#include "DetectorsRaw/HBFUtils.h"
+
+namespace o2::mch::raw
+{
+
+DigitReader::DigitReader(const char* digitTreeFileName)
+  : mFile(digitTreeFileName)
+{
+  mFile.GetObject("o2sim", mTree);
+  mDigitBranch = mTree->GetBranch("MCHDigit");
+  mDigitBranch->SetAddress(&mDigits);
+}
+
+void DigitReader::readNextEntry()
+{
+  assert(mDigitsPerIR.empty());
+
+  mDigitBranch->GetEntry(mEntry);
+  mEntry++;
+
+  // get the first and last IRs used
+  // (could be easier to find out if we had a ROFRecord of some sort)
+  uint32_t firstOrbit = std::numeric_limits<uint32_t>::max();
+  uint32_t lastOrbit = 0;
+
+  for (auto d : (*mDigits)) {
+    o2::InteractionTimeRecord ir(d.getTimeStamp());
+    firstOrbit = std::min(firstOrbit, ir.orbit);
+    lastOrbit = std::max(lastOrbit, ir.orbit);
+  }
```

Comment:
This seems to be remnants of old code, at least first/lastOrbit are calculated but never used

Reply:
Indeed. Thanks for spotting. Will remove it